### PR TITLE
create database dynamic reader

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: tests/test_dynamic_settings_database.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+try:
+    from simple_settings.dynamic_settings.database_reader import (
+        Reader as DatabaseReader,
+        DatabaseOperations
+    )
+except ImportError:
+    skip = True
+
+
+@pytest.mark.skipif(skip, reason='Installed without redis')
```
Comment (drgarcia1986) [inline tests/test_dynamic_settings_database.py]: `database` or `SQLAlchemy` instead `redis`

Comment (jairhenrique) [inline tests/test_dynamic_settings_database.py]: :+1:

File: tests/test_dynamic_settings_redis.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
     skip = True
 
 
-@pytest.mark.skipif(skip, reason='Installed without redis')
+@pytest.mark.skipif(skip, reason='Installed without SQLAlchemy')
```
Comment (drgarcia1986) [inline tests/test_dynamic_settings_redis.py]: I think you lost something :joy:

File: simple_settings/dynamic_settings/database_reader.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from sqlalchemy import engine_from_config, Column, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
```
Comment (drgarcia1986) [inline simple_settings/dynamic_settings/database_reader.py]: In redis and consul reader I added a simple try..except to raise ImportError with a better message, you can do this in this file?

https://github.com/drgarcia1986/simple-settings/blob/master/simple_settings/dynamic_settings/redis_reader.py#L4

Comment (jairhenrique) [inline simple_settings/dynamic_settings/database_reader.py]: :+1:

**Comment by drgarcia1986** [comment]
Awesome :clap: :clap: :clap: 
Many thanks for this killer contribution!

**Pull request merged**